Makefile

```diff
@@ -54,6 +44,12 @@ manager: fmt vet
 clusterctl: fmt vet
 	go build -o bin/clusterctl ./cmd/clusterctl
 
+# Build the clusterctl-tools container used for generating example clusterctl yaml
+clusterctl-tools:
+ifeq "$(shell docker images -q clusterctl-tools)" ""
```

Comment:
Hi @andrewsykim,

This should be:

```makefile
ifeq (,$(strip $(shell docker images -q clusterctl-tools))
```

Also, I'm not sure it should be a Make `ifeq` statement since this will fail if anyone runs any target and Docker isn't present or running. Perhaps make it part of the recipe instead of part of the target. Please consider rewriting this as:

```makefile
clusterctl-tools:
	[ -n "$(docker images -q clusterctl-tools)" ] || docker build -t clusterctl-tools -f cmd/clusterctl/examples/tools/Dockerfile .
```